### PR TITLE
Add date scrubber component to map controls.

### PR DESCRIPTION
**Pull request by tague**
Added a date scrubber component to the map controls to allow a user to
move the data display on the map back and forwards in time.

1. Updated the map header creation to include the date scrubber controls.

2. Improved the overall styling to be more conformant with the design
specifications for the header.

3. Replaced the info button svg with Material Icons font code to simplify
implementation.

4. Added a special-case for Alaska state borders. Alaska borders get
lost with a "negative space" border. Added a slight grey (grey-200)
border around Alaska to preserve shape across time.

### CODE REVIEW

File: src/choropleth.ts [hunk 1]
```diff
@@ -169,7 +231,7 @@ function colorizeMap(trend) {
       colorScale = colorScaleSafety;
       break;
     default:
-      console.log(`Unknown trend: ${trend} set on map`);
+      console.log(`Unknown trend: ${selectedTrend} set on map`);
```
Comment (tildechris) [inline src/choropleth.ts:234]: Remove debug console.log.

File: src/choropleth.ts [hunk 2]
```diff
@@ -267,10 +327,38 @@ function drawLegend(color) {
...
 
   d3.select("div#map-legend-date").text(
-    dateRangeString(latestCountyData["latestDate"])
+    dateRangeString(dateList[selectedDateIndex])
   );
 }
 
+function setDateControlState() {
+  if (selectedDateIndex == 0) {
+    console.log("setDateControlState: A");
```
Comment (tildechris) [inline src/choropleth.ts:336]: Remove 3 instances of debug console.log.

**Review by tildechris — APPROVED** [review]
This works great!  Only comment that I have is that this shows a disadvantage of having a constant color scale range.  When interest was much higher earlier this year, most of the county is in the top of the color range, making regional comparisons at that time difficult.  But that's outside of the scope of this change.  I'll open a new issue to consider changes to the scale.

File: src/choropleth.ts (hunk 1 above)
Comment (tague) [reply src/choropleth.ts:234]: This is there to log an error condition. I could change it to console.error, but I think it should stay.

File: src/choropleth.ts (hunk 2 above)
Comment (tague) [reply src/choropleth.ts:336]: Done

File: src/choropleth.ts (hunk 1 above)
Comment (tildechris) [reply src/choropleth.ts:234]: Yep, I didn't see that.  LGTM.

**Pull request merged**